aks/locals.tf

```diff
@@ -1,23 +1,120 @@
+resource "random_string" "name" {
+  length  = 3
+  special = false
+  numeric = false
+  upper   = false
+}
+
 locals {
+  roxiepool = {
+    ultra_ssd         = false
+    node_os           = "ubuntu"
+    node_type         = "gp"
+    node_type_version = "v2"
+    node_size         = var.aks_roxie_node_size
+    single_group      = false
+    min_capacity      = 0
+    max_capacity      = 3
+    labels = {
+      "lnrs.io/tier" = "standard"
+      "workload"     = "roxiepool"
+    }
+    taints = []
+    tags   = {}
+  }
+
+  node_groups0 = {
+    thorpool = {
+      ultra_ssd         = false
+      node_os           = "ubuntu"
+      node_type         = "gp"      # gp, gpd, mem, memd, stor
+      node_type_version = "v2"      # v1, v2
+      node_size         = var.aks_thor_node_size
+      single_group      = false
+      min_capacity      = 0
+      max_capacity      = var.aks_thorpool_max_capacity
+      labels = {
+        "lnrs.io/tier" = "standard"
+        "workload"     = "thorpool"
+      }
+      taints = []
+      tags   = {}
+    },
+
+    servpool = {
+      ultra_ssd         = false
+      node_os           = "ubuntu"
+      node_type         = "gpd"
+      node_type_version = "v1"
+      node_size         = var.aks_serv_node_size
+      single_group      = false
+      min_capacity      = 1
+      max_capacity      = 3
+      labels = {
+        "lnrs.io/tier" = "standard"
+        "workload"     = "servpool"
+      }
+      taints = []
+      tags   = {}
+    },
+
+    spraypool = {
+      ultra_ssd         = false
+      node_os           = "ubuntu"
+      node_type         = "gp"
+      node_type_version = "v1"
+      node_size         = var.aks_spray_node_size
+      single_group      = false
+      min_capacity      = 0
+      max_capacity      = 6
+      labels = {
+        "lnrs.io/tier"  = "standard"
+        "workload"      = "spraypool"
+        "spray-service" = "spraypool"
+      }
+      taints = []
+      tags   = {}
+    }
+  }
+
+  hpccpool = {
+    ultra_ssd         = false
+    node_os           = "ubuntu"
+    node_type         = "gp"      # gp, gpd, mem, memd, stor
+    node_type_version = "v2"      # v1, v2
+    node_size         = var.aks_serv_node_size
+    single_group      = false
+    min_capacity      = 4
```

Comment:
Please verify this min_capacity value.  Is it really needed?  I am somewhat suspicious of it, if we have trimmed the HPCC cluster down at all then we may not need 4 nodes at minimum.